packages/orchestrator-ui-components/src/hooks/useQueryWithGraphql.ts

```diff
@@ -18,21 +21,26 @@ export const useQueryWithGraphql = <U, V extends Variables>(
 ) => {
     const { graphqlEndpointCore } = useContext(OrchestratorConfigContext);
     const { session } = useSessionWithToken();
-    const requestHeaders = {
-        authorization: session ? `Bearer ${session.accessToken}` : '',
-    };
 
     const graphQLClient = new GraphQLClient(graphqlEndpointCore);
 
+    const requestHeaders: GraphQLClientRequestHeaders = {
+        Authorization: session ? `Bearer ${session.accessToken}` : '',
+    };
+
     const fetchFromGraphql = async () =>
         // TS-Ignore because queryVars does not seem to be accepted by the client
         // The props in this useQueryWithGraphql-hook ensures queryVars is indeed related to the query
         // eslint-disable-next-line @typescript-eslint/ban-ts-comment
         // @ts-ignore
-        await graphQLClient.request(query, queryVars, requestHeaders);
+        await graphQLClient.request<U, V>(query, queryVars, requestHeaders);
```

Comment:
I notice that all pages using this hook now contains a `if error then console log` block. Maybe move that logic into this hook. That keeps the consumers of this hook more clean

Reply:
Done